cmake/Modules/FindSYCL.cmake

```diff
@@ -427,7 +427,6 @@ macro(SYCL_LINK_DEVICE_OBJECTS output_file sycl_target)
       OUTPUT ${output_file}
       DEPENDS ${object_files}
       COMMAND ${SYCL_EXECUTABLE}
-      -fsycl
```

Comment:
Why `-fsycl` needs to be removed? Should `-fpreview-breaking-changes -fsycl` link against `sycl-preview`?

Reply:
`-fsycl` is already contained in `SYCL_device_link_flags`.
Yes, the goal is to link against `sycl-preview`.

Reply:
Regarding these SYCL libs, I'd like to reuse PyTorch variable rather than hard coding it.